src/tutorial/store/Item.java

```diff
@@ -0,0 +1,50 @@
+
+
+public class Item {
+	
+	
+	public Item(String itemName, String description, String storeDetails, int price) {
+		super();
+		ItemName = itemName;
+		Description = description;
+		StoreDetails = storeDetails;
```

Comment:
storeDetails is not required for now. We can limit the scope to item name, description and price